helm/ph-ee-engine/templates/connector-mpesa.yaml

```diff
@@ -50,6 +50,8 @@ spec:
               value: "{{ .Values.mpesa.SPRING_PROFILES_ACTIVE }}"
             - name: "ZEEBE_BROKER_CONTACTPOINT"
               value: "{{ .Values.mpesa.zeebe_broker_contactpoint }}"
+            - name: "ZEEBE_INIT-TRANSFER_WAIT-TIMER"
```

Comment:
@danishjamal104 : Please refer to the pinned naming standardization guidelines in Slack channel regarding what case to use in which file.